tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -538,6 +540,10 @@ def fit(self,
             If True, a WeightedEnsembleModel will be fit in each stack layer.
             A weighted ensemble will often be stronger than an individual model while being very fast to train.
             It is recommended to keep this value set to True to maximize predictive quality.
+        num_cpus: int, default = auto
+            The total amount of cpus you want AutoGluon predictor to use.
+        num_gpus: int, default = auto
+            The total amount of gpus you want AutoGluon predictor to use.
```

Comment:
Explain auto,

`"auto"` instead of `auto`

Tell the user that they generally shouldn't need to set this value.